### DISCUSSION TITLE
JavaScript Embed Code Not Working

### DISCUSSION THREAD

**Discussion by hasaneks** [vercel/next.js#16747] [Help]
I added javascript with URL but working sometimes. I need to add between body tag.

How I can solve this problem?

**Comment by jamesmosier** [comment]
If you want to embed a third party script, take a look at the [google analytics](https://github.com/vercel/next.js/blob/canary/examples/with-google-analytics/pages/_document.js) example where the script is placed in the `pages/_document.js` file.
